code/modules/research/xenobiology/crossbreeding/transformative.dm

```diff
@@ -0,0 +1,172 @@
+/*
+transformative extracts:
+	apply a permanent effect to a slime and all of its babies
+*/
+/obj/item/slimecross/transformative
+	name = "transformative extract"
+	desc = "It seems to stick to any slime it comes in contact with."
+	icon_state = "transformative"
+	effect = "transformative"
+	var/effect_applied = SLIME_EFFECT_DEFAULT
+
+/obj/item/slimecross/transformative/afterattack(atom/target, mob/user, proximity)
+	if(!proximity || !isslime(target))
+		return FALSE
+	var/mob/living/simple_animal/slime/S = target
+	if(S.stat)
+		to_chat(user, "<span class='warning'>The slime is dead!</span>")
+	if(S.transformeffects & effect_applied)
+		to_chat(user,"<span class='warning'>This slime already has the [colour] transformative effect applied!</span>")
+		return FALSE
+	to_chat(user,"<span class='notice'>You apply [src] to [target].</span>")
+	do_effect(S, user)
+	S.transformeffects = effect_applied //S.transformeffects |= effect_applied
+	S.effectsapplied = 1 //S.effectsapplied++
+	qdel(src)
+
+/obj/item/slimecross/transformative/proc/do_effect(mob/living/simple_animal/slime/S, mob/user)
+	SHOULD_CALL_PARENT(TRUE)
+	if(S.transformeffects & SLIME_EFFECT_LIGHT_PINK)
+		for(var/spawner in GLOB.mob_spawners)
+			LAZYREMOVE(GLOB.mob_spawners[spawner], S)
+		S.master = null
+		GLOB.poi_list -= S
+	if(S.transformeffects & SLIME_EFFECT_METAL)
+		S.maxHealth = round(S.maxHealth/1.3)
+	if(S.transformeffects & SLIME_EFFECT_BLUESPACE)
+		S.verbs -= /mob/living/simple_animal/slime/proc/teleport
+	if(S.transformeffects & SLIME_EFFECT_PINK)
+		var/datum/language_holder/LH = S.get_language_holder()
+		LH.selected_language = /datum/language/slime
```

Comment:
Applying another slime core on something that already has this one will remove the old effect?
Applying bluespace, then yellow removes the teleport verb?

Reply:
> Applying another slime core on something that already has this one will remove the old effect?
Applying bluespace, then yellow removes the teleport verb?

yes

Reply:
Yes it does, this is one of the things I tested for - though I didn't try every single combination of slime, I did check that each of them loses the effect when a new extract is applied.